### DISCUSSION TITLE
Account?

### DISCUSSION THREAD

**Discussion by git900000** [nikoniko78/scripts-java-physion#42] [General]
Hi guys,

I need someone to create me an account (Username and password) and give the details.

Thanks,

Gitney
;)

**Comment by git900000** [comment]
Sorry, posted in wrong tab :/

**Reply by git900000** [reply]
Much better

**Reply by nikoniko78** [reply]
Please specify. If you are asking to get a Physion account created by someone else, I can't do that for you as it technically falls under hacking.

**Reply by git900000** [reply]
I apologize. I needed one to run an account so I can invite people and post things and announcements.

My bad.

**Comment by nikoniko78** [comment]
i can add you as a moderator ONLY IF YOU USE YOUR POWER RESPECTFULLY. this
is a community and i dont want anything bad.

**Reply by git900000** [reply]
I swear, on my... Er...

....
......

On my Membership (!?!?) I  WILL NOT abuse my power. You have my word.
:|

**Comment by nikoniko78** [comment]
Please note that your moderation will have certain restrictions, like
deleting repository safety checkers.

**Reply by git900000** [reply]
Thank you. I will use this power responsibly. *Shakes hand*

**Reply by git900000** [reply]
And, er.. What are security checkers? Are they, like, equivalent to security checkpoints in airports?

**Comment by nikoniko78** [comment]
makes sure that your pull requests arent changing too much things or are
malicious

**Reply by git900000** [reply]
Oh, cool!
